src/averageAges.js

```diff
@@ -37,7 +41,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenArray = withChildren
+    ? people.filter((x) =>
```

Comment:
It's better to use `p` instead of `x` in all the functions here as it is more clear that `p` stays for `person`)